agenta-web/.env

```diff
@@ -1,2 +1,4 @@
 NEXT_PUBLIC_AGENTA_API_URL=http://localhost
-NEXT_PUBLIC_FF=oss
\ No newline at end of file
+NEXT_PUBLIC_FF=oss
+TELEMETRY_TRACKING_ENABLED=true
+POSTHOG_SECRET_KEY=phc_hmVSxIjTW1REBHXgj2aw4HW9X6CXb6FzerBgP9XenC7
```

Comment:
can you please make sure that this does not overwrite anything in ee

Reply:
Sure, sure. Thank you!

Reply:
I think `ee` would be the one to overwrite the contents of `.env` in `oss`.